doc/SpecifyingApplications/CreatingProfileInstanceList/CreatingProfileInstanceList.md

```diff
@@ -60,6 +59,49 @@ This is a step by step cookbook for creating the ProfileInstanceList.
 * UUIDs comprised in the references have to match the ServiceList content (otherwise the implementers will fill wrong data into the response bodies of the generic representation requests).  
 
 
+### Instances of FileProfile  
+
+* Whenever your application is supposed to use one or several files for storing data, a corresponding number of instances of FileProfile is required.  
+* Just copy/paste the form prepared for completion or copy from existing ProfileInstanceLists of other applications.  
+* Finally, just delete the form that waits for completion.  
+
+**ProfileName**  
+* The ProfileName shall be set on "FileProfile".  
+
+**UUID**  
+* The UUID shall follow the [Structure of UUIDs](../../ElementsApplicationPattern/Names/StructureOfUuids/StructureOfUuids.md).  
+* The LayerID shall be set on "file".  
+* The ObjectType shall be set on "p" (Profile).  
+* The ApiSegment shall be set on "im" (management, which is individual to this application).  
```

Comment:
Same as mentioned in the action profile , 

This statement is contradicting with the concept in the Structure of uuid

Either the uuid-structure needs to be corrected 
Or
This line should be deleted

Reply:
Deleted this line.